src/utils/shallowEqual.js

```diff
@@ -1,27 +0,0 @@
-export default function shallowEqual(objA, objB) {
```

Comment:
The implementation is almost 100% identical https://github.com/acdlite/recompose/blob/e85c61eafd07a5306d2bad5872f475a8bc8bf6a4/src/packages/recompose/shallowEqual.js.